crates/ruff_linter/src/rules/flake8_pytest_style/rules/parametrize.rs

```diff
@@ -687,11 +715,34 @@ fn handle_single_name(checker: &mut Checker, expr: &Expr, value: &Expr) {
         expr.range(),
     );
 
-    let node = value.clone();
-    diagnostic.set_fix(Fix::safe_edit(Edit::range_replacement(
-        checker.generator().expr(&node),
-        expr.range(),
-    )));
+    diagnostic.set_fix(Fix::unsafe_edits(
+        Edit::range_replacement(checker.generator().expr(value), expr.range()),
+        // If `argnames` and all items in `argvalues` are single-element sequences,
+        // they all should be unpacked. Here's an example:
+        //
+        // ```python
+        // @pytest.mark.parametrize(("x",), [(1,), (2,)])
+        // def test_foo(x):
+        //     assert isinstance(x, int)
+        // ```
+        //
+        // The code above should be transformed into:
+        //
+        // ```python
+        // @pytest.mark.parametrize("x", [1, 2])
+        // def test_foo(x):
+        //     assert isinstance(x, int)
+        // ```
+        //
+        // Only unpacking `argnames` would break the test:
+        //
+        // ```python
+        // @pytest.mark.parametrize("x", [(1,), (2,)])
+        // def test_foo(x):
+        //     assert isinstance(x, int)  # fails because `x` is a tuple, not an int
+        // ```
+        unpack_single_element_items(checker, argvalues),
```

Comment:
Should I add a test to ensure this doesn't conflict with PT007 which may also fix `argvalues`?

Reply:
This sounds like a great idea if we're concerned about whether the rules play nicely together.

Reply:
@MichaReiser Manually confirmed PT006 and PT007 don't conflicts:


```python
> cargo run -p ruff -- check --no-cache --select PT007,PT006 a.py --unsafe-fixes --fix
--- a.py
+++ a.py
@@ -1,6 +1,6 @@
 import pytest
 
 
-@pytest.mark.parametrize(("param",), [[1], [2], [3]])
+@pytest.mark.parametrize("param", [1, 2, 3])
 def test_foo(param):
     ...

Would fix 1 error.
```

Reply:
Added a test case for this.